posthog/api/feature_flag.py

```diff
@@ -460,7 +460,7 @@ def my_flags(self, request: request.Request, **kwargs):
             raise exceptions.NotAuthenticated()
 
         feature_flags = (
-            FeatureFlag.objects.filter(team=self.team, active=True, deleted=False)
+            FeatureFlag.objects.filter(team=self.team, deleted=False)
```

Comment:
in a follow up, you should also replace L479 with `get_all_feature_flags`, because right now this gives incorrect responses for some flags because it's missing logic for hash key overrides, for example.